### PR TITLE
* Use shallow clone when cloning on the CI

### PR DESCRIPTION
**Pull request by pranavkm**
* Avoid downloading the .build directory

cc @victorhurdugaci

### CODE REVIEW

File: makefile.shade [hunk 1]
```diff
@@ -98,7 +110,15 @@ var buildTarget = "compile"
                     blockLogger.StartBlock(blockName);
                 }
 
-                Log.Info(blockName);
+                if (IsLinux)
+                {
+                    Exec("copy", "-ar .build " + Path.Combine(repo, ".build"), "");
```
Comment (dougbu) [inline makefile.shade]: `No command 'copy' found, did you mean: ...` where none of the suggestions is the correct `cp`.

Comment (victorhurdugaci) [inline makefile.shade]: I think sake already has a copy shade file

Comment (davidfowl) [inline makefile.shade]: Yea, why isn;t this just using copy

Comment (pranavkm) [inline makefile.shade]: It uses File.Copy which is surprisingly slow when you have a bunch of files.

**Comment by pranavkm** [comment]
Updated

File: makefile.shade (hunk 1 above)
Comment (dougbu) [inline makefile.shade]: @pranavkm if you don't want to fix that in this PR, please file a follow-up to update `shade\_copy.shade` in Korebuild. We don't use the Sake script but do use `File.Copy()`.

**Pull request merged**